### ISSUE TITLE
api

### ISSUE THREAD

**Issue by amirtariq3** [darryldecode/laravelshoppingcart#319]
how can i create api of add cart.  because when i send items into cart then it does not show when i get it

**Comment by MostafaHamedAbdelmasoud** [comment]
yeah i changed to Database storage and still got nothing, I do believe you need more customization
